payjoin/src/send/error.rs

```diff
@@ -318,6 +318,22 @@ impl ResponseError {
             Err(_) => InternalValidationError::Parse.into(),
         }
     }
+
+    /// Whether the error provided by the payjoin receiver is suggesting a specific version of
+    /// payjoin.
+    ///
+    /// This is useful for when you support multiple versions of payjoin. For example, if you
+    /// support v1 and v2(BiP78 && BiP77 respectively), you can check if the error is suggesting a
+    /// specific version and if it does, you can retry the request based on the asked version.
```

Comment:
If we're going to spend the time to document functions let's write them with complete sentences and conventional grammar.

```suggestion
    /// Check whether or not the payjoin receiver supports the supplied version.
    ///
    /// This is useful when you support multiple versions of payjoin. For example, if you
    /// support v1 and v2 (BIP78 and BIP77 respectively) and sent a v2 request that
    /// resulted in this error, you could pass `1` to test whether or not the receiver might
    /// respond well to another request using the older version.
```

Reply:
thanks, changed.